include/z64message.h

```diff
@@ -82,6 +82,70 @@ typedef enum {
     /* 0x37 */ MSGMODE_PAUSED // Causes the message system to do nothing until external code sets a new message mode or calls a public function
 } MessageMode;
 
+typedef enum MaskReactionSet {
+    /* 0x00 */ MASK_REACTION_SET_CARPENTER_BOSS,
+    /* 0x01 */ MASK_REACTION_SET_CARPENTER_1,
+    /* 0x02 */ MASK_REACTION_SET_CARPENTER_2,
+    /* 0x03 */ MASK_REACTION_SET_CARPENTER_3,
+    /* 0x04 */ MASK_REACTION_SET_CARPENTER_4,
+    /* 0x05 */ MASK_REACTION_SET_HYRULIAN_GUARD,
+    /* 0x06 */ MASK_REACTION_SET_HEISHI4_1,
```

Comment:
all the other values in this enum use a friendly name, how come this one is still the OG actor name?

Reply:
idk, heishi4_1 and _2 are really `MASK_REACTION_SET_HYRULIAN_GUARD` _2 and _3